### DISCUSSION TITLE
Single quote used to get escaped/converted to &#039;

### DISCUSSION THREAD

**Discussion by snorkrat** [GrapesJS/grapesjs#4974] [Q&A]
I'm pretty sure when adding in text that contained single quotes/apostrophes, it used to get converted to` &#39; ` but no longer does.  Firstly, just want confirmation that this used to be the case, and if so is there a way I can enable this again?  I am using the webpage demo.

Thanks!

**Comment by artf** [comment]
Yeah it was actually changed [here](https://github.com/GrapesJS/grapesjs/commit/4b1bd3b71fc82713a2dd4463220a4cccbfef5e0c). You can update the `textnode` component and extend `__escapeContent` with your own escape strategy.
